modules/builds-troubleshooting-access-resources.adoc

```diff
@@ -10,6 +10,8 @@ If your request for access to resources is denied:
 Issue::
 A build fails with:
 
+.Example output
```

Comment:
Do we need the .Example output here? Is the output an actual string, not an example??

Reply:
Fixed - thanks!